recipes/pointpats/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pointpats" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3aaceb5576b8a587e5b8b81877dfd0b87439a199e8a813af27086effbd9ed673
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
```

Comment:
I think you have to set `python >=3`
![image](https://user-images.githubusercontent.com/7359284/57562947-c9b06480-734c-11e9-898a-758328a0c716.png)